docs/2_getting_started_session.md

```diff
@@ -207,8 +207,30 @@ dds user add [Email address] --role "Researcher" --project "<Project ID>"
 
 ###### Sub-section taks
 
- - [ ] TASK: Release the project after making sure all the data was uploaded, remember to specify the correct project ID
+ - [ ] TASK: Release the project after making sure all the data was uploaded - remember to specify the correct project ID
```

Comment:
I think we should let them download the data without releasing the project, specifically to emphasise on that feature.
The releasing action should be part of the reasearcher-account-with-a-separate-token-file task

Reply:
Ah yes, forgot to remove that text after removing the releasing from this